plugins/seedwatch.cpp

```diff
@@ -284,9 +298,12 @@ DFhackCExport command_result plugin_onstatechange(color_ostream &out, state_chan
 {
     switch (event) {
     case SC_MAP_LOADED:
+        if(running)
+            out.print("seedwatch supervision started.\n");
```

Comment:
This does result in a duplicate message when `seedwatch start` is placed in `onLoad.init`; other plugins don't tend to log their state when a world is loaded, so I'll probably remove this one.